examples/routing_service/routing_service_constrained_wan/localhostdemo/localhost.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0"?>
+<!--
+ (c) 2005-2020 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ RTI grants Licensee a license to use, modify, compile, and create derivative
+ works of the Software.  Licensee has the right to distribute object form only
+ for use with RTI products.  The Software is provided "as is", with no warranty
+ of any type, including any warranty for fitness for any purpose. RTI is under
+ no obligation to maintain or support the Software.  RTI shall not be liable for
+ any incidental or consequential damages arising out of the use or inability to
+ use the software.
+ -->
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+     xsi:noNamespaceSchemaLocation="http://community.rti.com/schema/6.0.1/6.0.0/rti_routing_service.xsd">
```

Comment:
Not sure why this file exists. Anyway this should be pointing to : http://community.rti.com/schema/6.0.1/rti_routing_service.xsd

Reply:
or http://community.rti.com/schema/6.0.0/rti_routing_service.xsd

Reply:
I used the template from \rti_workspace\6.0.1\user_config\routing_service\USER_ROUTING_SERVICE.xml which uses this path, and this file seems to exist on community. If this is incorrect then there's a bigger issue since it is being distributed with connext pro in USER_ROUTING_SERVICE.xml
I asked with AntonioS, but considering i'm following what our distributed software is doing I don't think it is a bug with my xml.

Reply:
Based on ROUTING-692 it is a bug so I'll change mine